src/validators/model.rs

```diff
@@ -62,18 +62,24 @@ impl BuildValidator for ModelValidator {
 
         let py = schema.py();
         for (key, value) in fields_dict.iter() {
-            let (validator, field_dict) = match build_validator(value, config, build_context) {
+            let field_infos: &PyDict = value.cast_as()?;
+            let schema: &PyAny = match field_infos.get_as("schema")? {
```

Comment:
you should be able to use `get_as_req` here and save some code.

Reply:
great!